docs/docs/Sprint 1/Arquitetura de Solucao/Requisitos Não Funcionais.md

```diff
@@ -5,12 +5,11 @@
 
 | Requisito Não Funcional | Descrição | Teste | Critério de aceitação |
 | ----------------------- | --------- | ----- | --------------------- |
-| RNF01 | O sistema deve garantir que os dados de temperatura coletados pelo robô sejam transmitidos para o backend através do ROS (Robot Operating System). O intervalo entre cada envio de dados deve ser estritamente de 5 minutos, com uma tolerância de ±1 minuto. | Durante um período de 30 minutos, monitore os intervalos entre cada transmissão de dados e anote o tempo exato de cada transmissão. Em seguida, calcule o intervalo real entre cada transmissão, subtraindo o tempo de transmissão atual do tempo de transmissão anterior. Por fim, verifique se o intervalo real entre cada transmissão de dados está na faixa especificada de 4 a 6 minutos. | 1. Os dados de temperatura devem ser enviados do robô para o backend através do ROS em intervalos regulares de 5 minutos. <br/> 2. A tolerância para o intervalo entre os envios de dados é de ±1 minuto, o que significa que o intervalo real pode variar entre 4 e 6 minutos. |
+| RNF01 | O sistema deve garantir que os dados coletados pelo robô sejam transmitidos para o backend através do ROS (Robot Operating System). O intervalo entre cada envio de dados deve ser estritamente de 5 minutos, com uma tolerância de ±1 minuto. | Durante um período de 15 minutos, monitore os intervalos entre cada transmissão de dados e anote o tempo exato de cada transmissão. Em seguida, calcule o intervalo real entre cada transmissão, subtraindo o tempo de transmissão atual do tempo de transmissão anterior. Por fim, verifique se o intervalo real entre cada transmissão de dados está na faixa especificada de 4 a 6 minutos. | 1. Os dados de temperatura devem ser enviados do robô para o backend através do ROS em intervalos regulares de 5 minutos. <br/> 2. A tolerância para o intervalo entre os envios de dados é de ±1 minuto, o que significa que o intervalo real pode variar entre 4 e 6 minutos. | 
```

Comment:
aqui está falando sobre dados de temperatura ainda

Reply:
Eu tirei a especificação de que seria de temperatura e deixei mais genérico: dados.
Dados de status do robô, dados das imagens que ele for fotografando... mas ok, posso especificar!